test/e2e/network/netpol/network_policy.go

```diff
@@ -217,6 +186,25 @@ var _ = SIGDescribeCopy("Netpol [LinuxOnly]", func() {
 			ValidateOrFail(k8s, model, &TestCase{FromPort: 81, ToPort: 80, Protocol: v1.ProtocolTCP, Reachability: reachability})
 		})
 
+		ginkgo.It("should enforce policy to allow all traffic to an application, having a deny policy [Feature:NetworkPolicy] ", func() {
```

Comment:
As I understand it, the point of this test case is to verify that given a deny-all and an explicit allow for the same target, the allow wins -- do I have that right?

Can that be clarified in the test description?  Trying to think of a more elegant way to state that :)

Reply:
Maybe moving the having a deny all traffic policy to a Ginkgo.By() clause in the test.
```
Ginkgo.By("having a deny Ingress policy", func() {
   policy = DenyPolicy() 
   CreatePolicy()
    ...
}) 
```